zeebe/exporter-api/src/main/java/io/camunda/zeebe/exporter/api/context/Context.java

```diff
@@ -30,6 +31,12 @@ public interface Context {
    */
   Logger getLogger();
 
+  /**
+   * @return a clock that provides the current time. Use this instead of system time to ensure that
```

Comment:
:question: Do we want to make it explicit that this is not related to the exporter's `Controller` task scheduling capabilities? I think it might cause confusion.

Reply:
I'll add something :+1: